### ISSUE TITLE
Is there something like chrome://flags for min?

### ISSUE THREAD

**Issue by ergpopler** [minbrowser/min#1457]
when i go to chrome://gpu, it gives me details, some settings aren't enabled, is there a way to enable them similar to chromium or no?

**Comment by ergpopler** [comment]
I do understand that i can use command line options, but some dont work, so it would be cool if there is a list or some documentation of what works and what doesn't

**Comment by Syndamia** [comment]
> is there a way to enable them similar to chromium or no

Nope, simply because Min isn't based on Chromium, it's based on Electron; which ties into 

> i can use command line options, but some dont work

you should look up for electron CLI options.
